src/enhanced_orchestrator.py

```
@@ -946,4 +427,384 @@ async def _upload_long_form_video(self, generation_result: Dict[str, Any]) -> Di
+        try:
+            if self.gpu_manager:
+                self.gpu_manager.cleanup()
+            
+            # Clean up other components
+            for component in [self.cinematic_editor, self.advanced_audio_processor, 
+                            self.enhanced_thumbnail_generator, self.enhancement_optimizer]:
+                if component and hasattr(component, 'cleanup'):
+                    try:
+                        await component.cleanup()
+                    except:
+                        pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)